core/services/bulletprooftxmanager/bulletprooftxmanager.go

```diff
@@ -277,12 +287,12 @@ func (b *BulletproofTxManager) CreateEthTransaction(db *gorm.DB, fromAddress, to
 	value := 0
 	err = postgres.GormTransactionWithDefaultContext(db, func(tx *gorm.DB) error {
 		res := tx.Raw(`
-INSERT INTO eth_txes (from_address, to_address, encoded_payload, value, gas_limit, state, created_at, meta, subject)
+INSERT INTO eth_txes (from_address, to_address, encoded_payload, value, gas_limit, state, created_at, meta, subject, pipeline_task_run_id, min_confirmations)
```

Comment:
I actually think it might make sense to override the meta field for this. This completes the abstraction a bit better since then all the domain-specific logic is on the side of the caller - it supplies custom meta and a custom callback function that can understand it.

We do get FK support this way though so I don't mind too much.